.github/coding_guidelines/CODING_GUIDELINES.md

```diff
@@ -0,0 +1,76 @@
+# MetaMask Coding Guidelines
+
+### 1. New Code Should be Typescript
+- New components and utilities should be written in Typescript and enforce typing.
+- Existing code should be refactored into Typescript where time allows. If you are replacing a component use Typescript.
```

Comment:
```suggestion
- Existing code should be refactored into Typescript where time allows. If you are replacing a component, use Typescript.
```

Reply:
Fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/20651/commits/725877d0181a3bdf6b965a6c5aab017824b7f728)